.github/workflows/test.yml

```diff
@@ -84,6 +84,52 @@ jobs:
             -destination 'platform=iOS Simulator,name=iPhone 8' \
             -skip-testing GPSTests/COVIDSafePathsTests/testRendersWelcomeScreen
         working-directory: ./ios
+  unit-test-ios-bt:
+    runs-on: macOS-latest
+    needs: unit-test
+    timeout-minutes: 25
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Set XCode Version
+        run: sudo xcode-select -s /Applications/Xcode_11.5.app
+
+      - name: Cache node_modules/
+        uses: actions/cache@v1
+        id: cache
+        with:
+          path: node_modules
+          key: ${{ runner.OS }}-yarn-cache-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.OS }}-yarn-cache-
+
+      - name: Install Node Dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: yarn --frozen-lockfile
+
+      - name: Cache ruby gems
+        uses: actions/cache@v1
+        with:
+          path: ios/vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - name: Install ruby gems (including Cocoapods)
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+        working-directory: ./ios
+
+      - run: pod install --repo-update
+        working-directory: ./ios
+
+      # TODO(https://pathcheck.atlassian.net/browse/SAF-250): remove the skip on COVIDSafePathsTests/testRendersWelcomeScreen once it's passing
```

Comment:
Would it help visibility to add this todo into Trello? Or maybe even the Jira board in case someone has the bandwidth to take it on?